client/modules/core/helpers/apps.js

```diff
@@ -6,44 +6,17 @@ import { Reaction } from "/client/api";
 import { Packages, Shops } from "/lib/collections";
 import { Registry } from "/lib/collections/schemas/registry";
 
-
 /**
- *
- * reactionApps
- *   provides="<where matching registry provides is this >"
- *   enabled=true <false for disabled packages>
- *   context= true filter templates to current route
- *   returns matching package registry objects
- *   @example {{#each reactionApps provides="settings" name=packageName container=container}}
- *   @example {{#each reactionApps provides="userAccountDropdown" enabled=true}}
- *   @example
- *     {{#each reactionApps provides="social" name="reaction-social"}}
- *         {{> Template.dynamic template=template data=customSocialSettings }}
- *     {{/each}}
- *
- *   @typedef optionHash
- *   @type {object}
- *   @property {string} name - name of a package.
- *   @property {string} provides -purpose of this package as identified to the registry
- *   @property {string} container - filter registry entries for matching container.
- *   @property {string} shopId - filter to only display results matching shopId, not returned
- *   @property {string} template - filter registry entries for matching template
- *   @type {optionHash}
- *
- *  @return {optionHash} returns an array of filtered, structure reactionApps
- *  [{
- *   enabled: true
- *   label: "Stripe"
- *   name: "reaction-stripe"
- *   packageId: "QqkGsQCDRhg2LSn8J"
- *   priority: 1
- *   provides: "paymentMethod"
- *   template: "stripePaymentForm"
- *   etc: "additional properties as defined in Packages.registry"
- *   ...
- *  }]
+ * @method Apps
```

Comment:
I don't know that this is actually a template helper? We use it in templates but I think we use it elsewhere.

Reply:
Would `reactionApps` be a better fit in `Helpers`?